File: app/src/main/java/com/example/neon/deardiary/util/CustomTimePickerDialog.java
package com.example.neon.deardiary.util;

import android.app.TimePickerDialog;
import android.content.Context;

/**
 * 解决onTimeSet()重复执行
 * Created by Neon on 2016/12/5.
 */

public class CustomTimePickerDialog extends TimePickerDialog {
    public CustomTimePickerDialog(Context context, OnTimeSetListener listener, int hourOfDay, int minute, boolean is24HourView) {
        super(context, listener, hourOfDay, minute, is24HourView);
    }

    @Override
    protected void onStop() {
//        super.onStop();
    }
}

File: app/src/main/java/com/example/neon/deardiary/widget/CustomTimePickerDialog.kt
package com.example.neon.deardiary.widget

import android.app.TimePickerDialog
import android.content.Context

/**
 * 解决 onTimeSet() 重复执行
 * Created by Neon on 2016/12/5.
 */

internal class CustomTimePickerDialog(context: Context, listener: TimePickerDialog.OnTimeSetListener, hourOfDay: Int, minute: Int, is24HourView: Boolean) : TimePickerDialog(context, listener, hourOfDay, minute, is24HourView) {

    override fun onStop() {
        //super.onStop();
    }
}
